hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/client/ConfiguredRMFailoverProxyProvider.java

```diff
@@ -25,8 +25,8 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import org.slf4j.Logger;
```

Comment:
why modifying this file?

Reply:
No particular reason, I made the  changed the other proxy providers to use the slf4j logger for consistency